src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/CommonLogger.cs

```diff
@@ -11,12 +13,14 @@ namespace NuGet.Services.Metadata.Catalog.Monitoring
     /// </summary>
     public class CommonLogger : Common.ILogger
     {
-        public CommonLogger(ILogger logger)
+        private static EventId logEvent = new EventId(23847);
+
+        public CommonLogger(Microsoft.Extensions.Logging.ILogger logger)
```

Comment:
This entire class is fully and easily unit testable.  Please add thorough unit tests for this class.